feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/README.md

```diff
@@ -2,24 +2,56 @@
 
 ## Testcase summary
 
-Ensure that protocol and DSCP based VRF selection is configured correctly.
+Test different VRF selection policies.
 
-## Testcase procedure
+## Procedure
+
+*   Connect ATE Port-1 to DUT Port-1, and ATE Port-2 to DUT Port-2.
+
+*   Configure DUT with ingress interface as Port-1 and egress interface as Port-2. DUT Port-2 is connected to ATE Port-2 and is configured for VLAN-10 and VLAN-20 such that sub-interfaces Port-2.10 and Port-2.20 are part of VLAN 10 and VLAN 20 respectively.
+
+*   Configure network-instance “VRF-10” of type L3VRF and assign Port2.10 to network-instance "VRF-10" 
+        
+*   Configure network-instance “DEFAULT” of type DEFAULT_INSTANCE. DUT-Port1 and DUT Port-2.20 should be part of "DEFAULT" Network instance by default w/o requiring explicit configuration.
+
+*   Configure Network-instance “VRF-10” with a default route pointing at ATE Port-2.10
```

Comment:
how about to be more specific? e.g. "with needed routes pointing"

Reply:
changed the test to instead point at ATE's directly connected IPv4 address.

Reply:
Changed back to using actual prefixes advertised by IXIA in place of IXIA interfaces